plugins/Plugin Map/admin/graphql/queries.ts

```diff
@@ -0,0 +1,33 @@
+import { gql } from '@apollo/client';
+
+export const GET_PLUGIN_MAP_REQUESTS = gql`
+  query GetPluginMapRequests($input: GetPluginMapRequestsInput) {
+    plugin_map_getPluginMapRequests(input: $input) {
+      requests {
```

Comment:
Why is the parent directory not `/src/pluginMap` ?

Reply:
Sorry, I meant /src/plugins/pluginMap similar to what you have in the test directory structure